crates/topos-tce-storage/src/errors.rs

```diff
@@ -33,6 +33,9 @@ pub enum InternalStorageError {
     #[error("Invalid query argument: {0}")]
     InvalidQueryArgument(&'static str),
 
+    #[error("Unexpected DB state: {0}")]
+    UnexpectedDBState(&'static str),
```

Comment:
It's not really a problem with the DB *state* if a property is missing. Wouldn't this error be for a situation where there's a new/old/incompatible rocksdb version that does not support some property?